aiohttp/client.py

```diff
@@ -271,7 +271,13 @@ def __init__(
         if timeout is sentinel or timeout is None:
             self._timeout = DEFAULT_TIMEOUT
         else:
-            self._timeout = timeout
+            if not isinstance(timeout, ClientTimeout):
+                raise ValueError(
+                    f"timeout parameter cannot be of {type(timeout)} type, "
+                    "please use 'timeout=ClientTimeout(...)'",
+                )
+            else:
+                self._timeout = timeout
```

Comment:
I feel like it might be simpler like:
```
        if timeout is sentinel or timeout is None:
            timeout = DEFAULT_TIMEOUT

        if not isinstance(timeout, ClientTimeout):
            raise ValueError(
                f"timeout parameter cannot be of {type(timeout)} type, "
                "please use 'timeout=ClientTimeout(...)'",
            )

        self._timeout = timeout
```

Reply:
yep, you totally right ^_^